Jellyfin.Plugin.Bookshelf/Providers/ComicInfo/ExternalComicInfoProvider.cs

```diff
@@ -0,0 +1,109 @@
+
+        private readonly ILogger<ExternalComicInfoProvider> _logger;
+
+        private const string ComicRackMetaFile = "ComicInfo.xml";
+
+        private ComicInfoXmlUtilities _utilities = new ComicInfoXmlUtilities();
+
+        public ExternalComicInfoProvider(IFileSystem fileSystem, ILogger<ExternalComicInfoProvider> logger)
+        {
+            this._logger = logger;
+            this._fileSystem = fileSystem;
+        }
+
+        public async Task<MetadataResult<Book>> ReadMetadata(ItemInfo info, IDirectoryService directoryService, CancellationToken cancellationToken)
+        {
+            var comicInfoXml = await this.LoadXml(info, directoryService, cancellationToken);
+
+            if (comicInfoXml is null)
+            {
+                this._logger.LogInformation($"Could not load ComicInfo metadata for ${info.Path} from XML file.");
+                return new MetadataResult<Book> { HasMetadata = false };
+            }
+
+            var book = this._utilities.ReadComicBookMetadata(comicInfoXml);
+
+            //If we found no metadata about the book itself, abort mission
+            if (book is null)
+            {
+                return new MetadataResult<Book> { HasMetadata = false };
+            }
+
+            var metadataResult = new MetadataResult<Book> { Item = book, HasMetadata = true };
+
+            //We have found metadata like the title
+            //let's search for the people like the author and save the found metadata
+            this._utilities.ReadPeopleMetadata(comicInfoXml, metadataResult);
+
+            this._utilities.ReadCultureInfoInto(comicInfoXml, "ComicInfo/LanguageISO", (cultureInfo) => metadataResult.ResultLanguage = cultureInfo.ThreeLetterISOLanguageName);
+
+            return metadataResult;
+        }
+
+        public bool HasItemChanged(BaseItem item, IDirectoryService directoryService)
+        {
+            var file = this.GetXmlFilePath(item.Path);
+            return file.Exists && _fileSystem.GetLastWriteTimeUtc(file) > item.DateLastSaved;
+        }
+
+        private async Task<XDocument?> LoadXml(ItemInfo info, IDirectoryService directoryService, CancellationToken cancellationToken)
+        {
+            var path = GetXmlFilePath(info.Path).FullName;
+
+            if (path is null)
+            {
+                return null;
+            }
+
+            try
+            {
+                // Open the xml
+                var reader = XmlReader.Create(path, new XmlReaderSettings { Async = true });
+                var comicInfoXml = XDocument.LoadAsync(reader, LoadOptions.None, cancellationToken);
+                // Read data from XML
+                return await comicInfoXml;
+            }
+            catch (Exception e)
+            {
+                _logger.LogInformation(e, $"Could not load external xml from {path}. This could mean there is no seperate ComicInfo metadata file for this comic. Maybe the metadata is bundled within the comic.", null);
```

Comment:
```suggestion
                _logger.LogInformation(e, "Could not load external xml from {Path}. This could mean there is no seperate ComicInfo metadata file for this comic. Maybe the metadata is bundled within the comic", info.Path);
```

Just a guess as to what the parameter should be